docs/de/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,216 @@
+# How to Firefox in German
+
+Dieses Dokument soll die Erstellung und Lokalisierung von Texten erleichtern – für optimal aufbereitete und konsistente Inhalte. Wichtig hierfür sind:
+
+* [eine zielgruppengerechte und markenspezifische Ansprache](Persönlichkeit & Tonalität)
+* [durchgängig geregelte Schreibweisen](Glossar)
+
+Das Dokument spiegelt den aktuellen Stand wieder. Unsere Marke entwickelt sich aber immer weiter, sodass Aktualisierungen weiterhin vorgenommen werden.
+
+**Warum ist dieser Guide wichtig für die Lokalisierung unserer Texte?**
+
+Unsere Texte sollen nicht nur eins zu eins übersetzt werden, sondern unsere Markenpersönlichkeit und Tonalität widerspiegeln und für den kulturellen und linguistischen Hintergrund optimiert werden. Die emotionale Intention des Ausgangstextes soll adaptiert werden.
+
+Unsere [Markeneigenschaften](Unsere Markeneigenschaften/Firefox Brand Personality) bestimmen dabei die Tonalität der englischen als auch deutschen Fassung. Dieser Guide soll dabei helfen, Firefox als Marke besser zu verstehen, um ein besseres Gefühl dafür zu bekommen, wie die Texte in der deutschen Sprache klingen sollen. Außerdem teilen wir hier festgelegte, unternehmensspezifische Schreibweisen, die den Lokalisierungsprozess erleichtern sollen.
+
+## About Firefox – wer wir sind
+
+Firefox ist mehr als nur der Browser. Firefox ist eine Challenger-Brand, die als Herausforderer den Status quo infrage stellt und sich für die Online-Rechte der Nutzer stark macht.
+
+Bei allem was wir tun, stellen wir die Rechte und Bedürfnisse der User an die erste Stelle. Und als Tech-Unternehmen stellen wir Menschen immer über Profit.
+
+Firefox als Marke vereint den Firefox Ethos und die sich erweiternde Bandbreite der Produkte.
+
+(Mozilla ist dabei die Non-Profit-Organisation, die hinter Firefox steht.)
+
+**Brand promise:**
+* Firefox fights for you
```

Comment:
I don't think the bullet point is really needed here? Same for the next section.